src/styles/content.scss

```diff
@@ -6,3 +9,10 @@ ol.os-raise-noindent {
 .os-raise-indent {
   padding-left: 2rem;
 }
+
+.os-raise-graybox {
```

Comment:
If we want to codify the limitation / expectation that this is for `<div>` we can make this `div.os-raise-graybox` so if it's included on any other element it'll get ignored.